### ISSUE TITLE
Size of the displayed items

### ISSUE THREAD

**Issue by andreacocci95** [RyanSusana/elepy#91]
When I add big images and a lot of text on an item, the preview show all the text and all the images. Can you fix this problem and set a maximum height for an item row?

![image](https://user-images.githubusercontent.com/22175753/71907206-d602f080-316b-11ea-888f-cc5ae811c75f.png)

**Comment by RyanSusana** [comment]
Hi,

What I can do is add a link that opens up a modal to show HTML content

**Comment by andreacocci95** [comment]
You mean that in the page of the screenshot, I see everything except for the html? Right?

**Comment by RyanSusana** [comment]
Yeah that's HTML content that gets rendered. My approach would be to render that in a modal
